### DISCUSSION TITLE
Support createTime/updateTime for DocumentSnapshot?

### DISCUSSION THREAD

**Discussion by anhnch** [invertase/react-native-firebase#6394] [General]
As I work with firebase-admin package. I see the DocumentSnapshot has 2 properties createTime and updateTime. Maybe Firestore generates them automatically. But react-native-base currently doesn't support. For now, I have to create 2 fields in documents and manage them on my own.

Of course it's better to have full control of the values on my own, but it's not very convenient to have them in almost all documents. I think it'll be great if invertase support them.

**Comment by mikehardy** [comment]
Hey there, have you checked firebase-ios-sdk or firebase-android-sdk APIs documentation to see if it's available?

**Reply by anhnch** [reply]
They don't include it in the [official doc](https://firebase.google.com/docs/reference/ios/firebasefirestore/api/reference/Classes/FIRDocumentSnapshot)  
It looks like they [don't expose them](https://github.com/firebase/firebase-ios-sdk/search?q=createTime&type=code) too
This [feature request](https://github.com/firebase/firebase-ios-sdk/issues/8641) did mention about the createTime and updateTime
